content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Phoenix: An end-to-end AI powered Social Media analysis platform powering peacebuilding
+efforts"
+date: "2025-02-10T10:16:57Z"
+description: "Phoenix is an open-source AI platform developed by Build Up and datavaluepeople to
+aid peacebuilding through social media analysis. Launched in January 2025, it addresses challenges
+like large data processing, limited technical expertise, and under-resourced-language
+classification. Key features include a user-friendly interface, secure authentication, data
+scraping, and multilingual classification. Phoenix has already supported hundreds peacebuilders,
+processed terabytes of data, and gained recognition at the Paris Peace AI Forum. A public-good
+platform, making advanced AI tools accessible for conflict analysis of digital spaces. Phoenix
```

Comment:
I think it should be "...digital spaces; Phoenix exemplifies..."

Reply:
b49b323